packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -48,6 +48,19 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
     toNumber
   )
 
+  async validatorSetAddresses(): Promise<string[]> {
```

Comment:
Not sure here... should we use a `getXXX()`  convention? or leave it as it is.

I leave that to you

Reply:
Let's keep with the convention.